### PR TITLE
Generate opam file, dune.3, omd.2.0.0~alpha3, emit tables!

### PR DESCRIPTION
**Pull request by MisterDA**
Emit tables using the odoc-parser "heavy" syntax (slightly easier than
the light syntax, similar to GitHub Flavored Markdown). Syntax for
alignment isn't yet defined.

https://github.com/ocaml-doc/odoc-parser/pull/11

### CODE REVIEW

**Comment by mseri** [comment]
Thanks a lot!

**Comment by MisterDA** [comment]
You could emit the "light" syntax instead which preserves alignment, but the code would be a tad more complex.
```
{t | title 1             | title 2 |
   |--------------------:|---------|
   |              a cell | cell    |
   | {e emphasized cell} | cell    | }
```

**Comment by mseri** [comment]
I don't mind either way. I think we can stick with this for the moment, and revise it once the syntax for alignment is finalized and released. What do you think?

File: tests/test.expected [hunk 1]
```diff
@@ -41,3 +41,4 @@ another code block?
 {!module:Module} <-- An explicit cross reference
 
 {!Module} <-- An inferred cross reference
+{table {tr {th foo}{th bar}{th tut}}{tr {td {e baz}}{td bim}{td [who]}}{tr {td per}{td {b lol}}{td dit}}}
```
Comment (mseri) [inline tests/test.expected:44]: Should we enforce that the table is isolated from the surrounding content by empty lines?

**Comment by mseri** [comment]
I will make a release once tables are supported by odoc as well

**Pull request merged**